session4/src/com/itschool/homework/Challenge1.java

```diff
@@ -0,0 +1,27 @@
+package com.itschool.homework;
+
+import java.sql.SQLOutput;
+
+public class Challenge1 {
+
+    public static void main(String[] args) {
+
+        int a = 1;
+        int b = 2;
+
+        System.out.println(a+b);
+        System.out.println(a-b);
+        System.out.println(a*b);
+        System.out.println(a/b);
+        System.out.println(a%b);
+
+        System.out.println("---");
+
+        int l = 10;
+        int L =2;
+        int A = l*L;
```

Comment:
I can understand what you mean here, but java convention for variable names is to start with lower case. You could have instead something like:
```
int length = 10;
int breadth = 2;
int area = length * breadth;
```